File: sdks/ruby/lib/novabilling/invoices/types/create_invoice_dto_status.rb
# frozen_string_literal: true

module Novabilling
  module Invoices
    module Types
      module CreateInvoiceDtoStatus
        extend Novabilling::Internal::Types::Enum

        DRAFT = "DRAFT"
        PENDING = "PENDING"
        PAID = "PAID"
        FAILED = "FAILED"
        CANCELED = "CANCELED"
      end
    end
  end
end

File: sdks/php/src/Invoices/Types/CreateInvoiceDtoStatus.php
<?php

namespace NovaBilling\Invoices\Types;

enum CreateInvoiceDtoStatus: string
{
    case Draft = "DRAFT";
    case Pending = "PENDING";
    case Paid = "PAID";
    case Failed = "FAILED";
    case Canceled = "CANCELED";
}
